nowcasting_api/database.py

```diff
@@ -336,74 +393,103 @@ def get_latest_forecast_values_for_a_specific_gsp_from_database(
     return forecast_values
 
 
-def get_session():
-    """Get database settion"""
-
-    with db_conn.get_session() as s:
-        yield s
-
-
 def get_latest_national_forecast_from_database(session: Session) -> Forecast:
-    """Get the national level forecast from the database"""
+    """
+    Get the national level forecast from the database.
 
-    logger.debug("Getting latest national forecast")
+    Parameters
+    ----------
+    session : Session
+        SQLAlchemy session
 
+    Returns
+    -------
+    Forecast
+        Latest national forecast
+    """
+    logger.debug("Getting latest national forecast")
     forecast = get_latest_national_forecast(session=session)
     logger.debug(forecast)
     return Forecast.from_orm(forecast)
 
 
+# GSP Yield / Truth Value functions
 def get_truth_values_for_a_specific_gsp_from_database(
     session: Session,
     gsp_id: int,
-    regime: Optional[str] = "in-day",
+    regime: str = "in-day",
     start_datetime: Optional[datetime] = None,
     end_datetime: Optional[datetime] = None,
 ) -> List[GSPYield]:
-    """Get the truth value for one gsp for yesterday and today
-
-    :param session: sql session
-    :param gsp_id: gsp id
-    :param regime: option for "in-day" or "day-after"
-    :param start_datetime: optional start datetime for the query.
-     If not set, after now, or set to over three days ago
-     defaults to N_HISTORY_DAYS env var, which defaults to yesterday.
-    :param end_datetime: optional end datetime for the query.
-    :return: list of gsp yields
     """
+    Get the truth value for one GSP for yesterday and today.
 
-    start_datetime = get_start_datetime(start_datetime=start_datetime)
-
-    return get_gsp_yield(
-        session=session,
-        gsp_ids=[gsp_id],
-        start_datetime_utc=start_datetime,
-        end_datetime_utc=end_datetime,
-        regime=regime,
+    Parameters
```

Comment:
why did we move it from `get_gsp_yield`

Reply:
I intended to unify parameter naming across API and DB layers by moving the `regime` argument into the route handlers. That keeps all formatting together, just like our other GSP endpoints, without altering functionality. Let me know if you have any concerns!